qkrehdwns/likelion-assignment/src/main/java/likelion/likelionassignment/member/api/dto/request/PostUpdateRequestDto.java

```diff
@@ -0,0 +1,7 @@
+package likelion.likelionassignment.member.api.dto.request;
```

Comment:
PostUpdateRequestDto의 위치가 member 아래에 있는 이유가 있을까요??
위치가 확인 부탁드려요!

```suggestion
package likelion.likelionassignment.post.api.dto.request;
```

Reply:
위치 수정하겠습니다!